pkg/auth/model/validation.go

```diff
@@ -3,20 +3,17 @@ package model
 import (
 	"errors"
 	"fmt"
-	"regexp"
 
 	"github.com/aws/aws-sdk-go/aws/arn"
 	"github.com/treeverse/lakefs/pkg/permissions"
 )
 
 var (
 	ErrValidationError = errors.New("validation error")
-
-	EntityIDRegexp = regexp.MustCompile(`^[\w+=.,@\-]{1,127}$`)
 )
 
 func ValidateAuthEntityID(name string) error {
-	if !EntityIDRegexp.MatchString(name) {
+	if len(name) == 0 {
 		return ErrValidationError
```

Comment:
maybe give more details?

```suggestion
	if len(name) == 0 {
		return fmt.Errorf("empty name: %w", ErrValidationError)
```